### PR TITLE
Enables groovy scripting

### PR DESCRIPTION
**Pull request by marcinc**
http://www.elastic.co/guide/en/elasticsearch/reference/current/modules-scripting.html

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -34,7 +34,8 @@ if [[ $# -lt 1 ]] || [[ "$1" == "--"* ]]; then
     -Des.default.bootstrap.mlockall \
     -Des.http.port=${ES_HTTP_PORT} \
     -Des.default.path.home=/usr/share/elasticsearch \
-    -Des.default.path.conf=/etc/elasticsearch "$@"
+    -Des.default.path.conf=/etc/elasticsearch "$@" \
+    -Des.script.groovy.sandbox.enabled=true
```
Comment (danharvey) [inline run.sh]: Maybe put this as a config option? We probably don't want that for eslog for example?

Comment (marcinc) [inline run.sh]: Fixed.

**Comment by danharvey** [comment]
+1

I've updated https://github.com/EqualMedia/service-units/pull/180 to enable this.

File: run.sh [hunk 2]
```diff
@@ -34,7 +35,8 @@ if [[ $# -lt 1 ]] || [[ "$1" == "--"* ]]; then
     -Des.default.bootstrap.mlockall \
     -Des.http.port=${ES_HTTP_PORT} \
     -Des.default.path.home=/usr/share/elasticsearch \
-    -Des.default.path.conf=/etc/elasticsearch "$@"
+    -Des.default.path.conf=/etc/elasticsearch "$@" \
```
Comment (vaijab) [inline run.sh]: You probably want to move `"$@"` to the end.

Comment (marcinc) [inline run.sh]: Will do.

**Pull request merged**